custom/opm/beneficiary.py

```diff
@@ -564,6 +564,24 @@ def weight_grade_normal(self):
                     return self.child_age/12
                 return False
 
+    def weight_grade_status(self, status):
+        if self.status == 'pregnant':
+            return False
```

Comment:
Here is the reason I re-added this. Consider an edge case where Child is born on Jan 15. And a CFU form is submitted on Jan 25, with some value for nutritional status. This case will be considered pregnant, but this particular function will still count this case into one of nutritional indicators. By adding this condition, we can filter out all such edge cases. Does that sound right?